themes/bootstrap3/templates/Recommend/Databases.phtml

```diff
@@ -0,0 +1,21 @@
+<div class="databases">
+  <?php $databases = $this->recommend->getResults(); ?>
+  <?php if (!empty($databases)): ?>
+    <h4><?=$this->transEsc('databases')?></h4>
```

Comment:
Any particular reason you chose an h4 here? A quick glance reveals some weird inconsistencies in our recommendation module heading levels, so I imagine you copied this from some existing template, but maybe it's not the best option -- it may be a whole separate project to straighten that out. (I notice, for example, that SummonResultsDeferred has an h3 that changes into an h4 when SummonResults is loaded... that can't be intentional).

Reply:
Yeah I don't recall which template I copied this from, but it looks like the majority (12/17) of recommendation headers are <h4>.  I agree there should be a separate PR to sort this out, and to figure out what heading level they should actually be.

![image](https://github.com/vufind-org/vufind/assets/31278545/7ef43f64-ac0d-4bca-98a3-7c64def6125c)

Reply:
I can see how the `h2` in SideFacets is usually the topmost heading, and then some of the other `h3`'s might be seen as subdivisions of that. But I really don't see how we get to `h4` at all. In any case, though, it is fine to keep this as-is to follow the predominant example, and we can straighten it out later. I've added a TODO as a reminder to open a follow-up after this is done. (I'm happy to do something to get the ball rolling when the time comes).

Reply:
Yeah whatever heading level it ends up after that other PR with is fine.